core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -137,9 +134,6 @@
       new JacksonJsonProvider();
   private static final MappingProvider JSON_PATH_MAPPING_PROVIDER =
       new JacksonMappingProvider();
-  private static final PrettyPrinter JSON_PRETTY_PRINTER =
```

Comment:
The same. We added this in https://github.com/apache/calcite/commit/6c9557fb554cd9e7178987030681982f2e85308e to prevent system-dependent behavior of the pretty printer.